client/src/actions/products.js

```diff
@@ -0,0 +1,16 @@
+import axios from 'axios'
+
+const LOAD_PRODUCTS = 'LOAD_PRODUCTS'
+
+export const loadProducts = products => ({
```

Comment:
just a note on `function declaration` and `function expression` https://gomakethings.com/function-expressions-vs-function-declarations/